NomaiGrandPrix/SpawnPoints.tsv

```diff
@@ -45,109 +59,95 @@ Spawn_DreamZone_3_SecretGallery	DreamZone 3 Secret Gallery	DreamZone	TRUE	FALSE
-Spawn_IP_Prison	Stranger Prison	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Ship	Stranger Ship	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Ship_OutsideCloak	Stranger Ship Outside Cloak	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_1	Stranger River Lowlands	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_1_BeachedRaft	Stranger River Lowlands Beached Raft	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_1_Rapids	Stranger River Lowlands Rapids	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_1_SecretPool	Stranger River Lowlands Secret Pool	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_2	Stranger Cinder Isles	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_2_CoveBottom	Stranger Cinder Isles Cove Bottom	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_2_CoveTop	Stranger Cinder Isles Cove Top	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_2_LighthouseTop	Stranger Cinder Isles Lighthouse Top	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_3_Bridge	Stranger Hidden Gorge Bridge	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_3_BrokenBridge	Stranger Hidden Gorge Broken Bridge	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_3_CodeDoor	Stranger Hidden Gorge Code Door	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_3_DreamFire	Stranger Hidden Gorge Dream Fire	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_3_DreamLabOverlook	Stranger Hidden Gorge DreamLab Overlook	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_3_Raft	Stranger Hidden Gorge Raft	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_3_SecretEntrance	Stranger Hidden Gorge Secret Entrance	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_3_Temple	Stranger Hidden Gorge Temple	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_4	Stranger Reservoir	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_4_LeftShore	Stranger Reservoir Left Shore	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_IP_Zone_4_RightShore	Stranger Reservoir Right Shore	Stranger	FALSE	FALSE	TRUE	FALSE	
-Spawn_JellyDomain	Giant's Deep Jelly Domain	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_GravityCannon	Gravity Cannon	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_TLE_Below	High Energy Lab Below	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_TLE_Interior	High Energy Lab Interior	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_TLE_Outside	High Energy Lab Outside	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+Spawn_TLEPath	High Energy Lab Path	EmberTwin	FALSE	TRUE	TRUE	FALSE	
+ShipSpawn_HT	Hourglass Twins (Orbit)	EmberTwin	FALSE	FALSE	FALSE	FALSE	
 Spawn_LakebedCave	Lakebed Cave	EmberTwin	FALSE	TRUE	TRUE	FALSE	
-SPAWN_MigrationPath	Migration Path		FALSE	TRUE	TRUE	FALSE	
-Spawn_Module_Broken	Orbital Probe Cannon Broken Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
-Spawn_Module_Intact	Orbital Probe Cannon Intact Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
-Spawn_Module_Sunken	Orbital Probe Cannon Sunken Module	GiantsDeep	FALSE	TRUE	TRUE	FALSE	
+Spawn_QuantumMaze	Quantum Maze (Wandering Rock)	EmberTwin	FALSE	TRUE	TRUE	FALSE	
```

Comment:
I think we should either call this Quantum Maze or Wandering Rock; I'd prefer to avoid parenthesis when possible.

Reply:
Game files called it Quantum Maze, I guess where that was but I thought that would clear it up, I could say "To Wandering Rock?